### PR TITLE
Update K8s URLs

### PR DESCRIPTION
**Pull request by yanivhershkovich**
The link specified next to the field names leads to 404 not found page.
Need to update with the correct link.

I just added "<<Need URL Update>>" next to each URL to point to it.
Remove that string once the URL is updated.

### CODE REVIEW

**Comment by yanivhershkovich** [comment]
Looks like my comment was truncated.
I added `<<Need URL Update>>` next to each URL that needs to be updated.
That string should be removed once the URL is updated

**Comment by bmansheim** [comment]
@alexmilowski Can you point these links to the operator.md file? It looks like it was moved.

File: content/platforms/kubernetes/cluster-options.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ You can create this binding manually, but we do not recommend it.
 
     {{< /note >}}
 
-- [`nodes`](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/docs/operator.md#redisenterpriseclusterspec): `nnn`
+- [`nodes`](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/docs/operator.md#redisenterpriseclusterspec) <<Need URL Update>>: `nnn`
```
Comment (alexmilowski) [inline content/platforms/kubernetes/cluster-options.md:56]: https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/redis_enterprise_cluster_api.md#redisenterpriseclusterspec

File: content/platforms/kubernetes/cluster-options.md [hunk 2]
```diff
@@ -62,7 +62,7 @@ You can create this binding manually, but we do not recommend it.
     The service_type must be either `ClusterIP` or `LoadBalancer` (default: `ClusterIP`).
     This is an optional configuration based on [k8s service types](https://kubernetes.io/docs/tutorials/kubernetes-basics/expose/expose-intro/).
 
-- [`persistentSpec`](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/docs/operator.md#persistentconfigurationspec):
+- [`persistentSpec`](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/docs/operator.md#persistentconfigurationspec) <<Need URL Update>>:
```
Comment (alexmilowski) [inline content/platforms/kubernetes/cluster-options.md:65]: https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/redis_enterprise_cluster_api.md#redisenterpriseclusterspec

File: content/platforms/kubernetes/cluster-options.md [hunk 3]
```diff
@@ -75,15 +75,15 @@ You can create this binding manually, but we do not recommend it.
       storageClassName: "gp2"
     ```
 
-- [`redisEnterpriseNodeResources`](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/docs/operator.md#redisenterpriseclusterspec):
+- [`redisEnterpriseNodeResources`](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/docs/operator.md#redisenterpriseclusterspec) <<Need URL Update>>:
```
Comment (alexmilowski) [inline content/platforms/kubernetes/cluster-options.md:78]: https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/redis_enterprise_cluster_api.md#redisenterpriseclusterspec

File: content/platforms/kubernetes/cluster-options.md [hunk 4]
```diff
@@ -75,15 +75,15 @@ You can create this binding manually, but we do not recommend it.
...
 
     The [compute resources](https://docs.openshift.com/enterprise/3.2/dev_guide/compute_resources.html#dev-compute-resources) required for each node (see `limits` and `requests`).
 
     {{< note >}}
 We recommend that resource limits equal requests ([Learn why](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/docs/topics.md#guaranteed-quality-of-service)).
     {{< /note >}}
 
-    - [`limits`](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/docs/operator.md#redisenterpriseclusterspec):
+    - [`limits`](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/docs/operator.md#redisenterpriseclusterspec) <<Need URL Update>>:
```
Comment (alexmilowski) [inline content/platforms/kubernetes/cluster-options.md:86]: https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/redis_enterprise_cluster_api.md#redisenterpriseclusterspec

File: content/platforms/kubernetes/cluster-options.md [hunk 5]
```diff
@@ -97,7 +97,7 @@ We recommend that resource limits equal requests ([Learn why](https://github.com
 
         The default is 4 cores (4000m) and 4GB (4Gi).
 
-    - [`requests`](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/docs/operator.md#redisenterpriseclusterspec):
+    - [`requests`](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/docs/operator.md#redisenterpriseclusterspec) <<Need URL Update>>:
```
Comment (alexmilowski) [inline content/platforms/kubernetes/cluster-options.md:100]: https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/redis_enterprise_cluster_api.md#redisenterpriseclusterspec

File: content/platforms/kubernetes/cluster-options.md [hunk 6]
```diff
@@ -111,7 +111,7 @@ We recommend that resource limits equal requests ([Learn why](https://github.com
 
         The default is 4 cores (4000m) and 4GB (4Gi).
 
-- [`serviceBrokerSpec`](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/docs/operator.md#servicebrokerspec):
+- [`serviceBrokerSpec`](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/docs/operator.md#servicebrokerspec) <<Need URL Update>>:
```
Comment (alexmilowski) [inline content/platforms/kubernetes/cluster-options.md:114]: This item needs to be deleted.

File: content/platforms/kubernetes/cluster-options.md [hunk 7]
```diff
@@ -121,7 +121,7 @@ We recommend that resource limits equal requests ([Learn why](https://github.com
          storageClassName: "*class_name*"
     ```
 
-- [`redisEnterpriseImageSpec`](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/docs/operator.md#imagespec):
+- [`redisEnterpriseImageSpec`](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/docs/operator.md#imagespec) <<Need URL Update>>:
```
Comment (alexmilowski) [inline content/platforms/kubernetes/cluster-options.md:124]: https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/redis_enterprise_cluster_api.md#imagespec

**Review by alexmilowski — CHANGES_REQUESTED** [review]

**Pull request merged**